recipes/pydarm/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "pydarm" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pydarm-{{ version }}.tar.gz
+  sha256: 82688a2e8b30dcf3ac761f09b0a1626cd3280cdcfaf26989933574695e41212f
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+    - setuptools-scm
+  run:
+    - corner
+    - dttxml
+    - emcee
+    - gwpy
+    - h5py
+    - matplotlib-base
+    - numpy
+    - python >=3.7
+    - requests
+    - scikit-learn
+    - scipy
+
+test:
+  imports:
+    - pydarm
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://pypi.org/project/pydarm/
+  summary: "'pyDARM modeling package for LIGO detector calibration'"
+  license: GPL-3.0
```

This statement is ambiguous, either it should be 

```suggestion
  license: GPL-3.0-only
```

or

```suggestion
  license: GPL-3.0-or-later
```